packages/nns/src/account_identifier.ts

```diff
@@ -58,6 +59,12 @@ export class AccountIdentifier {
   public toNumbers(): number[] {
     return Array.from(this.bytes);
   }
+
+  public toCandid(): AccountIdentifierCandid {
```

Comment:
wouldn't `toAccountIdentifier` be a more accurate name?

Reply:
It's already an accountIdentifier. But I can use `toAccountIdentifierHash` sounds good

Reply:
done